testapi.pm

```diff
@@ -1025,11 +1031,24 @@ Special characters naming:
 =cut
 
 sub send_key {
-    my ($key, $do_wait) = @_;
-    $do_wait //= 0;
-    bmwqemu::log_call(key => $key, do_wait => $do_wait);
-    query_isotovideo('backend_send_key', {key => $key});
-    wait_idle() if $do_wait;
+    my $key = shift;
+    my %args;
+    if (@_ == 1) {
+        %args = (do_wait => $_[0]);
+    }
+    else {
+        %args = @_;
+    }
```

Comment:
isn't this equivalent?
```perl
my %args = (@_ == 1) ? (do_wait => $_[0]) : @_;
```

Reply:
since i see you are not using @_ anymore, you could also:
```perl
my %args = (@_ == 1) ? (do_wait => shift) : @_;
```

Reply:
good idea, done. PR updated